ISISComputingGroup/ibex_gui
# Ticket2124: Delete multiple device screens

### Description of work

Allowed the user to select and delete multiple screens in the edit window.

### To test

https://github.com/ISISComputingGroup/IBEX/issues/2124

To test:
* Confirm that you can select multiple screens in the edit view.
* Confirm that the edit box is disabled when multiple screens are selected
* Confirm that you can delete when multiple screens are selected
* Confirm that pointing to a read only instrument you can only create local device screens

Note that there was quite a bit of refactoring here, general testing of the device screens view would be good.

---

#### Code Review


### Functional Tests


### Final Steps
